.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: python:2-stretch
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 17.10.0-ce
+      - run:
+          name: Install Docker client
+          command: |
+              set -x
+              curl -L https://get.docker.com/ | sh
```

Comment:
Why not use [their python-2 image](https://hub.docker.com/r/circleci/python/) so we get the latest docker and whatnot by default? `circleci/python:2`.

Reply:
Yes! Thank you